public/mock-smarti.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
@lobo87 This mock should actually be removed. Mocking Smarti should be done by injecting a Mock-Smarti-Adapter-implementation. I guess it could be done by replacing the [factory](https://github.com/assistify/Rocket.Chat/blob/develop/packages/dbs-ai/server/lib/Smarti.js#L89) with some class reading mock data from ... some other place ;)
Would just be nice if it wasn't hard coded.
Check with @tkurz what's state of the art mocking in JS

Reply:
ill have a talk with him

Reply:
@mrsimpson i have no smarti running locally atm, thats why i added this json. i moved the code in the according adapter and will remove the json file and adapt the mockservice to retreive data from localhost, if testing is enabled as soon as my local smarti is up and running